rotkehlchen/types.py

```diff
@@ -418,6 +418,15 @@ def is_bitcoin(self) -> bool:
     def is_substrate(self) -> bool:
         return self in get_args(SUPPORTED_SUBSTRATE_CHAINS)
 
+    def get_native_token_id(self) -> str:
+        """Returns the string identifier of the native token for the chain"""
+        if self == SupportedBlockchain.OPTIMISM:
+            return 'ETH'
+        if self == SupportedBlockchain.POLYGON_POS:
+            return 'eip155:137/erc20:0x0000000000000000000000000000000000001010'
+
```

Comment:
Should it use here

```suggestion
        if self == SupportedBlockchain.POLYGON_POS:
            return A_POLYGON_POS_MATIC.identifier

```

Reply:
no this is not possible. This is a very ugly and impossible to fix cyclic import. KEep in mind the file is types.py which is one of the most imported files in the project. 

We should take out huge classes like this from there.